src/main/java/net/snowflake/ingest/streaming/internal/ChannelData.java

```diff
@@ -56,6 +58,18 @@ public static Map<String, RowBufferStats> getCombinedColumnStatsMap(
     return result;
   }
 
+  /**
+   * Combines the two paris of min/max insert timestamp together
```

Comment:
Why is this needed? Why do we need to combine two pairs?

Reply:
because this is at chunk level, not channel level

Reply:
Got it, makes sense!